docs/docs/forms.md

```diff
@@ -389,7 +390,7 @@ class Form extends React.Component {
 
   handleChange(event) {
     let val = event.target.value;
-    let checked = this.state.checked.slice(); // copy
+    let checked = this.state.checked.slice(); // copy 
     if(checked.includes(val)) {
```

Comment:
Please put space between `if` and the opening brace.
Please don't use `includes()`, it's a ES2016 API that is unsupported by any other than most recent browsers.